### ISSUE TITLE
The "expand" type is described as using `help` by default, but instead the first choice is used by default

### ISSUE THREAD

**Issue by tjb1982** [CITGuru/PyInquirer#50]
I suppose this could be just a change to the readme to correctly describe the default behavior, but there is currently no way that I can find to make `help` the default for the `expand` type.

**Comment by CITGuru** [comment]
@tjb1982 I dont understand what you mean

**Comment by tjb1982** [comment]
I mean when you provide a list of choices and no default, like 

```python
{
  "type": "expand",
  #"default": "<=commented-out",
  "choices": [...],
}
```

The default (i.e., hitting the return key) should expand the help, but instead the first provided choice is the default.

**Comment by CITGuru** [comment]
Yes, you mean it should show up the help rather than the default if the default is not given.

**Comment by tjb1982** [comment]
I mean, that's what I thought the docs said it would do. I also think it should be possible to manually assign the help as the default, but that doesn't seem to work either.

**Comment by CITGuru** [comment]
Hmmnnn, lemme check that out.

**Comment by tjb1982** [comment]
If you accept PRs on this I can probably help out, also. Let me know.

**Comment by CITGuru** [comment]
Sure I accept PRS. I am actually looking for contributors now

**Comment by tjb1982** [comment]
Sorry for the delayed reply. Okay, great. I'm actively using this library on a project, so as I go, I'll make PRs. I'm side tracked right now, though, with a project at work, so it might be a couple of weeks.

**Comment by CITGuru** [comment]
Sure, no problem. Thanks
